.chloggen/rename_httpforwarder_extension.yaml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: deprecation
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: breaking
```

Component: breaking? 🤔  Shouldn't it be:
```suggestion
component: httpforwarder
```